app/models/company.rb

```diff
@@ -1,5 +1,13 @@
 class Company < ActiveRecord::Base
-  validates :name, presence: true
-  validates :name, uniqueness: true
-  has_many :jobs
+  validates :name, presence: true, uniqueness: true
+  has_many :jobs, dependent: :destroy
+  has_many :contacts
+
+  def self.top_three_companies_by_level_of_interest
```

Comment:
i don't see a test for this method - be sure to test all methods you write in your models!